lib/microbadger-helpers.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+function findVersionByTag(data, tag) {
+  let result;
+  if (!data.Versions) {
+    return;
+  }
+  for (let i = 0; !result && i < data.Versions.length; i++) {
+    const versionJson = data.Versions[i];
+    for (let j = 0; !result && j < versionJson.Tags.length; j++) {
+      const tagJson = versionJson.Tags[j];
+      if (tag === tagJson.tag) {
+        result = versionJson;
+      }
+    }
+  }
+  return result;
```

Comment:
If I'm understanding correctly, this would be equivalent:
```js
return data.Versions.find(v => v.Tags.some(t => t.tag === tag))
```

Since you only call it once, you could move that code to the handler and eliminate the helper.

Reply:
Refs:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Yes, you got it correctly. Thanks for the refs though!